core/src/scss/utilities/variables/core/_typography.scss

```diff
@@ -60,15 +60,15 @@ $su-intro-line-height: 1.5 !default;
 /// @name $su-font-sans
 ///
 /// @group variable
-$su-font-sans: 'Source Sans Pro', $font-stack-helvetica !default;
+$su-font-sans: 'Source Sans 3', 'Source Sans Pro', $font-stack-helvetica !default;
 
 ///
 /// font-serif font family.
 ///
 /// @name $su-font-serif
 ///
 /// @group variable
-$su-font-serif: 'Source Serif Pro', $font-stack-georgia !default;
+$su-font-serif: 'Source Serif 4', 'Source Serif Pro', $font-stack-georgia !default;
```

Comment:
The 2 places where we update the font stack is in the typography core variables. These are then referenced in the mixins and classes